### ISSUE TITLE
Nameserver question

### ISSUE THREAD

**Issue by davidhay45** [pavel-demin/red-pitaya-notes#777]
I am using Alpine SD card 3.6.  The redpitaya is hardwired to my pc which connects to the internet via wifi.

The gateway address on the rp is 192.168.1.1 but my wifi router ip is now 192.168.1.254 (as I changed isp).  I do not have internet access from the rp.  How do I change the gateway address and nameserver addresses?

**Comment by davidhay45** [comment]
Windows 10.  The change of router address might be a red herring as I don't know if the rp had internet access before.

**Comment by davidhay45** [comment]
Thanks for the idea but that won’t work in this set up.  If I enable internet connection sharing I lose all connectivity from my pc.

I can’t even ping the W10’s ip address pc from the RP so something more basic is wrong.

This is nothing to do with RP, so many thanks for even looking at it.

**Comment by davidhay45** [comment]
How do others work with wspr in respect of internet connections?  Is it possible to configure the RP with a wired connection to my pc for use with HPSDR etc and a wireless connection to my home router to allow the wspr to post the collected data to wsprnet.org?

I am not asking Pavel to tell me how to configure it just if it looks possible.

Thanks again

**Comment by davidhay45** [comment]
To finally conclude this, I tried to get the wifi and wired connections working together but without success.  The problem was that I had to use a wired connection between my pc and my Redpitaya to get a reliable fast connection but that meant that the RP had no connection to the internet.  My PC and RP are remote from my broadband router.

The solution was to use a wifi access point with an ethernet port to connect to an ethernet switch to which both the PC and RP are connected.  It is simple but was not obvious initially.
